selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java

```diff
@@ -76,22 +76,36 @@ public CommandsEditor(
     PageFactory.initElements(seleniumWebDriver, this);
   }
 
-  public static final class CommandsEditorType {
-    public static final String COMMAND_LINE_EDITOR =
-        "(" + Locators.ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[1]";
-    public static final String PREVIEW_URL_EDITOR =
-        "(" + Locators.ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[2]";
+  public enum CommandsEditorType {
+    COMMAND_LINE_EDITOR("(" + ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[1]"),
+    PREVIEW_URL_EDITOR("(" + ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[2]");
 
-    private CommandsEditorType() {}
+    private final String locator;
+
+    CommandsEditorType(String locator) {
+      this.locator = locator;
+    }
+
+    public String get() {
+      return this.locator;
+    }
   }
 
-  public static final class CommandsMacrosLinkType {
-    public static final String EDITOR_MACROS_LINK =
-        "(" + ACTIVE_EDITOR_ENTRY_POINT + "//a[@id='gwt-debug-link-explore_macros'])[1]";
-    public static final String PREVIEW_MACROS_LINK =
-        "(" + ACTIVE_EDITOR_ENTRY_POINT + "//a[@id='gwt-debug-link-explore_macros'])[2]";
+  public enum CommandsMacrosLinkType {
```

Comment:
Is it really about link to ONE macro? If so, lets fix enum and constant names to refer to ONE macro.

Reply:
![eclipse che wksp o3gh](https://user-images.githubusercontent.com/22724619/37960130-796a609c-31bd-11e8-9967-d4d7c6947d3c.png)

![eclipse che wksp o3gh 1](https://user-images.githubusercontent.com/22724619/37960134-7bf15cda-31bd-11e8-96ed-32c13ae9337f.png)

I think we should leave this class name